src/features/student/views/StudentUpdateAdditionalSkillView/components/AdditionalSkillLevelFormField/AdditionalSkillLevelFormField.test.ts

```diff
@@ -0,0 +1,118 @@
+import type { UpdateAdditionalSkillForm } from '@/features/student/views/StudentUpdateAdditionalSkillView/components/use-update-additional-skill-form/use-update-additional-skill-form'
+import AdditionalSkillLevelFormField
+  from '@/features/student/views/StudentUpdateAdditionalSkillView/components/AdditionalSkillLevelFormField/AdditionalSkillLevelFormField.vue'
+import { BddTest } from '@avenirs-esr/avenirs-dsav/test-utils'
+import { useForm } from '@tanstack/vue-form'
+import { mount, type VueWrapper } from '@vue/test-utils'
+import { beforeEach, expect, vi } from 'vitest'
+
+const TestWrapper = {
+  components: {
+    AdditionalSkillLevelFormField
+  },
+  setup () {
+    const form = useForm({
+      defaultValues: {
+        level: 'INTERMEDIATE'
+      },
+      validators: {
+        onSubmit ({ value }) {
+          return {
+            fields: {
+              level: !value.level
+                ? 'Ce champ est requis'
+                : undefined
+            }
+          }
+        }
+      }
+    }) as unknown as UpdateAdditionalSkillForm
+
+    return { form }
+  },
+  template: `
+    <form @submit.prevent="form.handleSubmit">
+      <AdditionalSkillLevelFormField :form="form" />
+    </form>
+  `
+}
+
+BddTest().given('an additional skill level form field component', () => {
+  let wrapper: VueWrapper
+
+  const stubs = {
+    AddAdditionalSkillLevelField: {
+      name: 'AddAdditionalSkillLevelField',
+      props: ['form', 'errorMessage'],
+      emits: ['blur'],
+      template: '<div><select @change="$emit(\'blur\')"><option value="BEGINNER">Débutant</option><option value="INTERMEDIATE">Intermédiaire</option></select><span v-if="errorMessage" class="error">{{ errorMessage }}</span></div>'
```

Comment:
```suggestion
      template: `
        <div>
          <select @change="$emit(\'blur\')">
            <option value="BEGINNER">Débutant</option>
            <option value="INTERMEDIATE">Intermédiaire</option>
          </select>
          <span
            v-if="errorMessage"
            class="error"
          >
            {{ errorMessage }}
          </span>
        </div>
      `
```

Je sais que c'est qu'un stub mais on peut rendre ça visuellement clair malgré tout ;)